picard/tagger.py

```diff
@@ -246,6 +246,10 @@ def __init__(self, method_name, help_text=None, help_args=None):
     ),
     "LOOKUP": RemoteCommand(
         "handle_command_lookup",
+        help_text="Lookup all unclustered files in the cluster pane.",
+    ),
+    "LOOKUP_CLUSTERS": RemoteCommand(
+        "handle_command_lookup_clusters",
```

Comment:
Does it lookup clusters or files in clusters? I think the `LOOKUP` command is unclear about what it does, and adding `LOOKUP_CLUSTERS` is even more confusing.

We could use a command with arguments instead: `LOOKUP <what>` with a default.

What do you think?

Reply:
We could do a single command with an argument.  I actually like that idea.  Perhaps the argument options could be:

- `clustered` - Lookup files that have been clustered.
- `unclustered` - Lookup files that have not been clustered.

Does that help clarify?

Reply:
I like this idea. Not sure about the default either, though. What about allowing `all` and make that the default?

Reply:
That sounds okay to me.  I'll make the change.